options/options.go

```diff
@@ -46,6 +46,13 @@ type TerragruntOptions struct {
 	// Current Terraform command being executed by Terragrunt
 	TerraformCommand string
 
+	// Original Terraform command being executed by Terragrunt. Used to track command evolution as terragrunt chains
```

Comment:
Will this be set to `apply-all` all the way through? Or switch to `apply` at some point?

Reply:
Good question! Added comment to clarify what happens for xxx-all commands.